themes/default/content/templates/container-service/aws/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Deploy an ECS Container Service on AWS
```

Comment:
```suggestion
title: Container Service on AWS
```

Reply:
the title format deviates here because we are trying to SEO rank for the term "deploy ecs"